test/actions/submit-drake.js

```diff
@@ -23,32 +23,28 @@ const correctCharacteristics = {
         wings: "No wings"
       },
       correctAlleles = "a:T,b:T,a:M,b:M,a:w,b:w,a:h,b:h,a:C,b:C,a:B,b:B,a:Fl,b:Fl,a:hl,b:hl,a:A1,b:A1,a:D,a:Bog,a:rh",
-      userCharacteristics = {
-        armor: "Five armor",
-        color:"Steel",
-        forelimbs:"Forelimbs",
-        health: "Healthy",
-        hindlimbs: "No hindlimbs",
-        horns: "Horns",
-        liveliness: "Alive",
-        "nose spike": "No nose spike",
-        tail: "Long tail",
-        wings: "Wings"
-      },
       userAlleles = "a:T,b:T,a:M,b:M,a:W,b:W,a:h,b:h,a:C,b:C,a:B,b:B,a:Fl,b:Fl,a:hl,b:hl,a:A1,b:A1,a:D,a:Bog,a:rh",
       initialAlleles = "a:T,b:T,a:M,b:M,a:W,b:w,a:h,b:h,a:C,b:C,a:B,b:B,a:Fl,b:Fl,a:hl,b:hl,a:A1,b:A1,a:D,a:Bog,a:rh",
+      motherAlleles = "a:T,b:T,a:M,b:M,a:W,b:w,a:h,b:h,a:C,b:C,a:B,b:B,a:Fl,b:Fl,a:hl,b:hl,a:A1,b:A1,a:D,a:Bog,a:rh",
+      fatherAlleles = "a:T,b:T,a:M,b:M,a:W,b:w,a:h,b:h,a:C,b:C,a:B,b:B,a:Fl,b:Fl,a:hl,b:hl,a:A1,b:A1,a:D,a:Bog,a:rh",
       getState = () => ({
         routeSpec: {level: 0,mission: 0, challenge: 0},
         drakes: [
           {
-            phenotype: {characteristics: correctCharacteristics},
             alleleString: correctAlleles,
             sex: 0
           },
           {
-            phenotype: {characteristics: userCharacteristics},
```

Comment:
I don't think `phenotype` is ever a part of the drakes array, so not sure why these need to be here. Maybe something in the past that was refactored. I removed these from the test, and it seems to pass fine.